cdap-ui/app/cdap/components/PipelineSummary/index.js

```diff
@@ -117,13 +118,19 @@ export default class PipelineSummary extends Component {
     this.eventEmitter = ee(ee);
   }
   componentWillReceiveProps(nextProps) {
+    if (isEqual(nextProps.totalRunsCount, this.props.totalRunsCount)) {
```

Comment:
Why do we need `isEqual` here instead of just using `===`? Also if we have this check then we don't need the if check on line 124.